rubin_sim/moving_objects/pre_generate.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from rubin_sim.data import get_data_dir
+import glob
+import os
+from rubin_sim.moving_objects import Orbits, DirectObs
+
+
+if __name__ == "__main__":
+    mjd_start = 60676.0
+    length = 365.25 * 12  # How long to pre-compute for
+    dtime = 1
+    mjds = np.arange(mjd_start, mjd_start + length, dtime)
+
+    orbit_files = glob.glob(os.path.join(get_data_dir(), "orbits/") + "*.txt")
+    output_dir = os.path.join(os.path.join(get_data_dir(), "orbits_precompute/"))
+
+    names = ["ra", "dec"]
+    types = [float] * 2
+    dt = list(zip(names, types))
+
+    for filename in orbit_files:
+        print("working on %s" % filename)
+        orbits = Orbits()
+        orbits.read_orbits(filename)
+        # Array to hold results
+        results = np.zeros((len(orbits.orbits), np.size(mjds)), dt)
+        do = DirectObs()
```

Comment:
May I suggest using PyOrbEphemerides directly to generate these ephemerides? That way you don't have to do any FOV checking or calculate colors or trailing losses. There is an example here - https://github.com/lsst/rubin_sim_notebooks/blob/main/moving_objects/moving_objects_demo_part_1.ipynb

Reply:
It's calling `generate_ephemerides` not `run`, so there is no extra checking or trailing losses calculated. I like using `DirectObs` because then I know these positions were created with the default kwargs like `obs_time_scale`, so as long as I'm using defaults later it will all match up.

Reply:
Reasonable. It might make sense to add some information like the timescale into the saved files, or at least some information in the README associated with those files about some of these details.